src/zone.rs

```diff
@@ -306,6 +368,50 @@ impl Zone {
         }
     }
 
+    // TODO factorize it with contains
```

Comment:
is this TODO still relevant ?

Reply:
I have no idea...

Reply:
I think it is, this method is very similar to the `contains` method (my bad :sweat_smile: )